sodym/mfa_system.py

```diff
@@ -74,53 +74,63 @@ def get_new_array(self, **kwargs) -> NamedDimArray:
         dims = self.dims.get_subset(kwargs["dim_letters"]) if "dim_letters" in kwargs else self.dims
         return NamedDimArray(dims=dims, **kwargs)
 
-    def get_relative_mass_balance(self):
-        """Determines a relative mass balance for each process of the MFA system.
-
-        The mass balance of a process is calculated as the sum of
+    def get_mass_balance(self):
+        """The mass balance of a process is calculated as the sum of
         - all flows entering subtracted by all flows leaving (-) the process
         - the stock change of the process
-
-        The total mass of a process is caluclated as the sum of
-        - all flows entering and leaving the process
-        - the stock change of the process
-
-        The process with ID 0 is the system boundary.
-        Its mass balance serves as a mass balance of the whole system."""
-
-        # start of with minimum possible dimensionality;
-        # addition and subtraction will automatically reduce to the maximum shape, i.e. the dimensions contained in all
-        # flows to and from the process
```

Comment:
I don't completely understand this comment. We start with 0, and try to add to it, so the dimensionality gets reduced -> under the hood we sum over all dimensions?

Reply:
- When we add the first flow to a scalar, the dimension set of the balance become that of the first flow
- Whenever we add a further flow to it, the dimension set of the balance becomes the set intersection of the current balance dimension and the added flow
- So in total, the dimensions will be the set intersection of all flows connected to that process, i.e. the dimensions that are common to all flows connected to that process. For example, if flows `A (t,e,r)`, `B(t,e,r,s)` and `C(t,e,w)` connect to a process, the dimensions of the balance will be `(t,e)`.

Reply:
Makes sense, thanks!